### PR TITLE
Bump PocoCpp to 1.7.4

### PR DESCRIPTION
**Pull request by shekharhimanshu**
Bump PocoCpp to [1.7.4](https://github.com/pocoproject/poco/releases/tag/poco-1.7.4-release)

### CODE REVIEW

File: cmake/projects/PocoCpp/hunter.cmake [hunk 1]
```diff
@@ -8,11 +8,11 @@ hunter_add_version(
     PACKAGE_NAME
     PocoCpp
     VERSION
-    1.7.3
+    1.7.4
```
Comment (ruslo) [inline cmake/projects/PocoCpp/hunter.cmake]: Please **add** new version, do not erase old. See example: https://github.com/ruslo/hunter/blob/master/cmake/projects/GTest/hunter.cmake. Thanks.

More details:
* https://docs.hunter.sh/en/latest/overview/what-is-it.html
* https://docs.hunter.sh/en/latest/overview/customization/config-id.html
* https://docs.hunter.sh/en/latest/overview/customization/hunter-id.html

Comment (shekharhimanshu) [inline cmake/projects/PocoCpp/hunter.cmake]: Ah, My bad! Will correct it. Thanks for those links.

**Comment by shekharhimanshu** [comment]
Updated!

**Comment by ruslo** [comment]
Testing:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.740
* https://travis-ci.org/ingenue/hunter/builds/148496806

**Comment by ruslo** [comment]
Uploading:
* https://travis-ci.org/ingenue/hunter/builds/148505076
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.743

**Comment by ruslo** [comment]
Released: https://github.com/ruslo/hunter/releases/tag/v0.15.4

**Comment by shekharhimanshu** [comment]
Thanks!

**Pull request merged**